doc/kubevela-definitions/cardano/node-connector.md

```diff
@@ -0,0 +1,109 @@
+# Cardano Node Connector trait
+
+## Overview
+
+The Cardano Node Connector in Kubvela acts as a bridge between your application and the Cardano node through specific application characteristics and settings
+
+The ***required*** parameter for implementing this trait is:
+ - `network`: Defines the network choice for the Cardano Node, with a default setting as "preview". 
+<br>The other options available are "preprod" and "mainnet".
+
+This configuration effectively attaches specific volumes to each container that is part of your application. Also, a `socat` container bridges your application and the Cardano Node by utilizing TCP Socket over a UNIX Socket.
```

Comment:
This is an implementation detail which may change, all the user needs to know is that there will be a volume with the node socket.